src/mantarray_desktop_app/mc_simulator.py

```diff
@@ -131,46 +145,80 @@ def _send_data_packet(
 
     def _commands_for_each_run_iteration(self) -> None:
         self._handle_test_comm()
+        # if _reboot_time_secs is not None, this means the simulator is in a "reboot" phase
+        if self._reboot_time_secs is not None:
+            secs_since_reboot = _get_secs_since_reboot_command(self._reboot_time_secs)
+            # if secs_since_reboot is less than the reboot duration, simulator is still in the 'reboot' phase. Commands from PC will be discared and status beacons will not be sent
+            if secs_since_reboot < 5:
```

Comment:
a note about this number `5` would be useful.  Right now, I remember that this comes from a conversation with Aster/Farshad about the maximum amount of time it would take to reboot a Mantarray, but if we look back at this code in a month we could easily forget why there's a `5` here.   i.e. whether it was something chosen due to queue multiprocessing timing issues, a semi-arbitrary number, or something specifically tied to the way the Mantarray hardware operates